site/source/static/css/_stats.css

```diff
@@ -0,0 +1,36 @@
+/* Stats
+--------------------------------------------------------------------------- */
+
+.statistiques-resume {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 30px;
+}
+
+.statistiques-resume > div {
+  flex: 1 0 auto;
+  text-align: center;
+  box-shadow: 0 1px 4px #c9d3df;
```

Comment:
Pour les couleurs il vaudrait mieux utiliser une des variables du fichier principal, ou bien même en ajouter une au besoin pour rester homogène sur tout le site : https://github.com/SocialGouv/work-in-france/blob/master/site/source/static/css/_base.css#L55-L76

Si tu as préparé des éléments réutilisables ça serait bien aussi de les ajouter au cookbook de façon à pouvoir les intégrer n'importe où dans le site. Comme ça quelqu'un peut s'en servir à nouveau plus tard https://workinfrance.beta.gouv.fr/cookbook.html